triggerdotdev/trigger.dev
# Added extraCACerts config

Closes #1210 

## ✅ Checklist


---

## Testing
1. Generate a self signed cert and use this to protect trigger.dev deployment
2. create a task which tries to access a self hosted api with self signed certificate
3. try to deploy the project to self-hosted instance, see it fail
4. create a ca.crt file containing signing certificate public key in pem format
5. copy this file to project dir
6. add `extraCACerts` config `./ca.crt`
7. deploy the task to self-hosted instance, this time it should deploy and run successfully
---

## Changelog
---
Added config option `extraCACerts` to `ProjectConfig` type. This copies the ca file along with `additionalFiles` and sets `NODE_EXTRA_CA_CERTS` environment variable in built image as well as running the task.

## Screenshots

_[Screenshots]_

💯
